www/pages/sell/new.tsx

```diff
@@ -122,20 +128,47 @@ function NewSellOrder() {
               />
             </label>
 
-            <label className="flex flex-1 flex-col">
+            <label className="flex-col">
               <div className="font-sans mb-1 text-base">Token</div>
+              <SelectSearch
+                className={(classes: string) => {
+                  return dropDownUI(classes) + ' w-40';
+                }}
+                options={optimismList}
+                placeholder={customTokenDisabled ? 'USDC' : 'Custom Token'}
+                onChange={(opt) => {
+                  // @ts-ignore
+                  if (opt === 'Custom') {
```

Comment:
Instead of doing `// @ts-ignore`, you probably want:

```ts
onChange={(opt: any) => {
```

And that should fix your types here